src/scene_server.py

```diff
@@ -7,19 +7,35 @@
 from fastapi import FastAPI
 from lynx.common.object import Object
 from lynx.common.scene import Scene
+from lynx.common.enitity import Entity
 from pydantic import BaseModel
 
 
 def execution_runtime(pipe: AioConnection, object_id: int):
     from lynx.common.actions.move import Move
     from lynx.common.vector import Vector
+    from lynx.common.actions.action import Action
+
+    scene_serialized = pipe.recv()
+    scene: Scene= Scene.deserialize(scene_serialized)
+
+    def send(action: Action):
+        pipe.send(action.serialize())
+        scene = pipe.recv()
+
+    builtins = {
+        'move': lambda vector: send(Move(object_id, vector)),
+        'Vector': Vector,
+    }
 
     while (True):
-        scene_serialzied = pipe.recv()
-        scene = Scene.deserialize(scene_serialzied)
-        action = Move(object_id=object_id, vector=Vector(10, 10))
-        time.sleep(1)
-        pipe.send([action.serialize()])
+        # Sure, I know exec bad
+        # pylint: disable=exec-used
+        exec(
+            scene.get_object_by_id(object_id).tick,
+            {'__builtins__': builtins},
+            {},
```

Comment:
Why empty brackets?

Reply:
It's an optional argument, so it seems to me you can skip it: https://docs.python.org/3/library/functions.html#exec:~:text=In%20all%20cases%2C%20if%20the%20optional%20parts%20are%20omitted%2C%20the%20code%20is%20executed%20in%20the%20current%20scope.

Reply:
Done, but I'm worried that we might forget sometime that we can use this dictionary for locals.